diff --git a/mk/Variables.mk b/mk/Variables.mk
index 99f787c..3ffd87a 100644
--- a/mk/Variables.mk
+++ b/mk/Variables.mk
@@ -42,12 +42,6 @@ CFLAGS		+= -Wall -Wmissing-prototypes -Wstrict-prototypes \
 		   -Wpointer-arith -Wunused
 CXXFLAGS	+= -Wall -Wpointer-arith -Wunused
 
-# add /usr/local to the search path if something is in there ...
-ifneq ($(wildcard /usr/local/include/*.h),)
-  CFLAGS  += -I/usr/local/include
-  LDFLAGS += -L/usr/local/$(LIB)
-endif
-
 # fixup include path for $(srcdir) != "."
 ifneq ($(srcdir),.)
   CFLAGS  += -I. -I$(srcdir)
